src/Main.hs

```diff
@@ -36,7 +38,7 @@ command str = let res = words str
                    [":help"] -> Help
                    _ -> Unknown
 
-helpMessage = "\n:load filename\t\tTo load the given filename\n"++
+helpMessage = "\n:load filename directory\t\tTo load the given filename with imports stored in directory\n"++
```

Comment:
Is 'directory' an optional parameter? If yes, it should be marked appropriately.